### DISCUSSION TITLE
I can't get education certification, and the email says I didn't authorize the browser, which is not true.

### DISCUSSION THREAD

**Discussion by SZYD717** [community/community#52370] [GitHub Education]
Reasons for refusal: Modify your browser settings to allow Github to use your location information. Once location is allowed you MUST cancel  your current application and begin again.

 I have applied for educational qualifications on multiple devices, using multiple browsers, and have failed more than 30 times.
The reason for the rejection was that I had not authorized location information to Github. However, I had set the location permissions for education.github.com in my browser, canceled my previous application, and logged back into Github. But it didn't help.
I can see in the Location Control panel for Windows 10 and Android that the browser gets my location information after I click the 'continue' button. But then I still get an email telling me that I didn't grant the browser location.

**Comment by samus-aran** [comment]
Hi @SZYD717 

We've had a number of members having success following these suggestions. 

- https://github.com/orgs/community/discussions/50578
- https://github.com/orgs/community/discussions/50261

Let us know how you get on 👍
